doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -3,17 +3,17 @@
 Using Ubuntu as the Service OS
```

Comment:
Should the title be updated to "Service VM"?

Reply:
Let's make that: "Running Ubuntu in the Service VM". Would that work for you?

Reply:
Yes! I think this is much better and more accurate. Thanks.